src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -81,7 +81,7 @@ struct buf
     if (!column_page_size)
       return;
 
-    auto pad = (page + 1) * column_page_size;
+    auto pad = static_cast<size_t>(page + 1) * column_page_size;
```

Comment:
Is this cast really necessary?  Was it added because of compiler complaining about possible roll over?
If so the cast does nothing to prevent that because it is applied after the roll over.